ansible/module_utils/graph_utils.py

```diff
@@ -22,6 +22,7 @@ class LabGraph(object):
         "console_links": "sonic_{}_console_links.csv",
         "bmc_links": "sonic_{}_bmc_links.csv",
         "l1_links": "sonic_{}_l1_links.csv",
+        "serial_links": "sonic_{}_serial_link.csv",
```

Comment:
"serial connection" is also console connection, correct?
If yes, why not reuse the existing "sonic_{}_console_links.csv file?